spec/lib/hyrax/hirmeos/metrics_tracker_spec.rb

```diff
@@ -59,17 +63,44 @@
         children: nil
       }
       work_json = tracker.work_to_hirmeos_json(work)
-      tracker.submit_to_hirmeos(work.id, work_json)
+      tracker.submit_works_to_hirmeos(work.id, work_json)
       expect(a_request(:post, tracker.translation_base_url + "/works").with(body: structure.to_json)).to have_been_made.at_least_once
     end
 
     it 'does not call the register endpoint if a work is already registered' do
       work_json = tracker.work_to_hirmeos_json(work)
-      tracker.submit_to_hirmeos(work.id, work_json)
+      tracker.submit_works_to_hirmeos(work.id, work_json)
       expect(a_request(:post, tracker.translation_base_url + "/works")).not_to have_been_made
     end
   end
 
+  describe '#submit_file_sets_to_hirmeos' do
+    it 'Makes a call to the translator works endpoint if the work is not already registered' do
+      path = "#{Hyrax::Hirmeos::MetricsTracker.translation_base_url}/works?uuid=#{hirmeos_uuid}"
+      stub_request(:get, path).to_return(status: 200, body: hirmeos_work_data[:data].to_json)
```

Comment:
This is great and so much easier to read than the other stub_requests in this test.  Do you think we could make `path` a let at the top of each describe block and replace `Hyrax::Hirmeos::MetricsTracker.translation_base_url` with `tracker.translation_base_url` where ever it occurs? `path` is also set in a before block, so we should remove that so it's clear for each example what value of path is being used.  Also `stub_request(:get, Addressable::Template.new(path))...` is called in that before block.  Is this needed?

Reply:
Yep, that's much clearer as you can see which path everything is accessing, and they are better named too!